sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-08-13)
+- Preview release of `azure-communication-networktraversal`.
+
+The first preview of the Azure Communication Relay Client has the following features:
+
+- get a relay configuration by creating a CommunicationRelayClient
+
+### Added
```

Comment:
This section should be named as `Features Added`. See [CHANGELOG format](https://azure.github.io/azure-sdk/policies_releases.html#change-logs).